src/sushi.sol

```diff
@@ -58,6 +63,7 @@ contract SushiJoin is CropJoin {
 
     MasterChefLike  immutable public masterchef;
     address                   public initialMigrator;
+    address                   public initialRewarder;
     TimelockLike              public timelockOwner;
     uint256                   public pid;
```

Comment:
Looks like this can't change on the MC side--any reason it can't be `immutable`?

Reply:
This can be changed via the `set()` function on MC with the `overwrite` flag set to true. https://github.com/sushiswap/sushiswap/blob/master/contracts/MasterChefV2.sol#L134

Reply:
That line you've highlighted is changing the `rewarder`, not the `pid`. I still don't see where the `pid` can change--as it's always an index into the `poolInfo` array.

Reply:
Oops yes you're right. Immutable it is.